### ISSUE TITLE
Terminal closes randomly upon pressing enter

### ISSUE THREAD

**Issue by musicformellons** [rcaloras/bashhub-client#32]
When using the terminal it randomly closes (so sometimes it performs the folder shift and sometimes just closes, could not find any pattern...). I tracked the problem down to bashhub additions to .bashrc

Here it is:
http://askubuntu.com/questions/790752/gnome-terminal-quits-when-pressing-enter?noredirect=1#comment1188726_790752

It is a pity because I really like bashhub's features.

**Comment by rcaloras** [comment]
@musicformellons Thanks for reporting. Noticed this randomly after upgrading to version 0.3.0 of bash-preexec. I believe this is caused by https://github.com/rcaloras/bash-preexec/issues/25

Could you check the bash and bashhub versions for me?
```bash
echo "$BASH_VERSION"
bashhub version
```

You can still use bashhub if you downgrade to 1.0.1 which doesn't use 0.3.0 of bash-preexec. Could you give it a try?
```bash
bashhub update 1.0.1
```

Going to work on fixing [bash-preexec](https://github.com/rcaloras/bash-preexec) and potentially downgrade the version in bashhub to 0.2.3.

**Comment by musicformellons** [comment]
bash version:
4.3.42(1)-release

bashhub version
Bashhub 1.0.2

Downgraded to 1.0.1 ; looks ok now!

**Comment by rcaloras** [comment]
@musicformellons Just released a dev version of bashhub to address this. https://github.com/rcaloras/bashhub-client/pull/33 It disables subshell support by default. Could you give it a quick try? 

Simply run:

```bash
bashhub update 1.1.1-dev
```
Let me know if you have any issues. Thanks!

**Comment by musicformellons** [comment]
Will try in two weeks (on vacation right now). Thanks.

**Comment by rcaloras** [comment]
@musicformellons this should be fixed in bashhub 1.1.1

`bashhub update` for the latest. 

Closing this, feel free to reopen if you see it's still a problem in the new version.

**Comment by musicformellons** [comment]
Sorry, forgot about it. Just updated and looks good. Thanks a lot!
